vue-frontend/src/components/jobs/CareerView.vue

```diff
@@ -22,7 +22,10 @@
       <img :src="loader" />
     </div>
     <div v-else class="career-container normal-text mt-5">
-      <div class="career-list-container">
+      <div v-if="careers.length == 0" class="text-center py-14 normal-text">
```

Comment:
Can you please explain this condition ??

Reply:
I have explained in MR description